packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.routes.js

```diff
@@ -11,22 +11,6 @@ export default /* @ngInject */ ($stateProvider) => {
       value: ['.'],
     },
     resolve: {
-      isKycFeatureAvailable: /* @ngInject */ ($http) => {
-        return $http
-          .get(`/feature/identity-documents/availability`, {
-            serviceType: 'aapi',
-          })
-          .then(
-            ({ data: featureAvailability }) =>
-              featureAvailability['identity-documents'],
-          );
-      },
-      kycStatus: /* @ngInject */ ($http, isKycFeatureAvailable) => {
-        if (isKycFeatureAvailable) {
-          return $http.get(`/me/procedure/identity`).then(({ data }) => data);
-        }
-        return false;
-      },
       needkyc: /* @ngInject */ (isKycFeatureAvailable, kycStatus) => {
         if (isKycFeatureAvailable) {
           return ['required', 'open'].includes(kycStatus.status);
```

Comment:
`needKyc` should also be removed, shouldn't it ? Because `isKycFeatureAvailable` is longer defined

Reply:
isKycFeatureAvailable is now coming from a parent state